packages/plugin-ext/src/plugin/command-registry.ts

```diff
@@ -26,9 +28,17 @@ export class CommandRegistryImpl implements CommandRegistryExt {
     private proxy: CommandRegistryMain;
     private commands = new Map<string, Handler>();
 
+    private readonly _converter: CommandsConverter;
```

Comment:
hello, as per https://github.com/theia-ide/theia/wiki/Coding-Guidelines#names variables with _ should not be used

Reply:
thank you for reminding me )
fixed